File: mathvar.cs
// datavar.cs
using System;

class Program
{
    static void Main()
    {
        int a = 5;
        int b = 3;
        int sum = a + b;
        int diff = a - b;
        int prod = a * b;
        int quot = a / b;

        Console.WriteLine("5 + 3 = " + sum);
        Console.WriteLine("5 - 3 = " + diff);
        Console.WriteLine("5 * 3 = " + prod);
        Console.WriteLine("5 / 3 = " + quot);
    }
}

File: mathvar.pas
{ datavar.pas }
program Datavar;
var
  a, b, sum, diff, prod, quot: integer;
begin
  a := 5;
  b := 3;
  sum := a + b;
  diff := a - b;
  prod := a * b;
  quot := a div b;
  
  writeln('5 + 3 = ', sum);
  writeln('5 - 3 = ', diff);
  writeln('5 * 3 = ', prod);
  writeln('5 / 3 = ', quot);
end.
